common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
This CoA should be a bit highter.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122671746-327b9380-d1d9-11eb-86a9-aeb6377be1a7.png)

</details>